packages/react-kit/src/components/Scrollable/Scrollable.styl

```diff
@@ -6,12 +6,13 @@
 	position: relative;
 }
 
-.withHorizontalScrollbar {
+.withHorizontalScrollbar:not(.contentUnderScrollbars) {
 	> .wrapper > .container > .content {
```

Comment:
Looks like this cascade can be replaced with direct passing of theme object to the underlying child component.